### PR TITLE
Feat timestamp attribute

### PR DESCRIPTION
**Pull request by lohanidamodar**
## What does this PR do?

Adds a support for Timestamp attributes in database collections

## Test Plan

Updated database tests

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

YES

### CODE REVIEW

File: app/controllers/api/database.php [hunk 1]
```diff
@@ -1004,6 +1005,71 @@ function createAttribute(string $collectionId, Document $attribute, Response $re
...
+    ->groups(['api', 'database'])
+    ->label('event', 'database.attributes.create')
+    ->label('scope', 'collections.write')
+    ->label('sdk.namespace', 'database')
+    ->label('sdk.auth', [APP_AUTH_TYPE_KEY])
+    ->label('sdk.method', 'createTimestampAttribute')
+    ->label('sdk.description', '/docs/references/database/create-timestamp-attribute.md')
+    ->label('sdk.response.code', Response::STATUS_CODE_CREATED)
+    ->label('sdk.response.type', Response::CONTENT_TYPE_JSON)
+    ->label('sdk.response.model', Response::MODEL_ATTRIBUTE_INTEGER)
```
Comment (Meldiron) [inline app/controllers/api/database.php:1019]: Timestamp here, right?

File: app/views/console/database/collection.phtml [hunk 2]
```diff
@@ -720,6 +723,71 @@ $logs = $this->getParam('logs', null);
...
+        <div class="row responsive thin">
+            <div class="col span-6 margin-bottom-small">
+                <label for="timestamp-min">Min</label>
+                <input id="timestamp-min" type="number" step="1" class="full-width" name="min" autocomplete="off" data-cast-to="integer" />
+            </div>
+            <div class="col span-6 margin-bottom-small">
+                <label for="timestamp-max">Max</label>
+                <input id="timestamp-max" type="number" step="1" class="full-width" name="max" autocomplete="off" data-cast-to="integer" />
+            </div>
+        </div>
```
Comment (Meldiron) [inline app/views/console/database/collection.phtml:774]: Min and max in UI for timestamp is number. Can we make it "date"?

File: app/views/console/database/collection.phtml [hunk 3]
```diff
@@ -720,6 +723,71 @@ $logs = $this->getParam('logs', null);
...
+
+        <label for="timestamp-default">Default Value</label>
+
+        <template x-if="!(array || required)">
+            <input name="xdefault" type="number" class="margin-bottom-large" autocomplete="off" data-cast-to="integer">
+        </template>
+        <template x-if="(array || required)">
+            <input name="xdefault" type="number" class="margin-bottom-large" autocomplete="off" data-cast-to="integer" disabled>
+        </template>
```
Comment (Meldiron) [inline app/views/console/database/collection.phtml:783]: Default in UI for timestamp is number. Can we make it "date"?

**Review by Meldiron — CHANGES_REQUESTED** [review]
left few comments

File: app/views/console/database/collection.phtml (hunk 2 above)
Comment (lohanidamodar) [reply app/views/console/database/collection.phtml:774]: Will look into it

**Pull request closed without merging**